config/cl.yml

```diff
@@ -68,8 +68,35 @@ entries:
 - prefix: /2011-12-13/
   replacement: https://cell-ontology.googlecode.com/svn/releases/2011-12-13/
 
+- regex: ^/releases/(201.*)/
```

Comment:
We don't use the `regex` system very often. The examples in the README all end with `$` but I don't think that's what you want here.

I *think* you could get what you want with a series of `prefix` rules.

Reply:
Also, it's conflicting with the new rule for release.

Reply:
This is still failing the tests.

Entries are processed in order from top to bottom, which is why I think that a series of `prefix` entries will do what you want.

Reply:
Will it work to only change the key `regex` to `prefix`?